### DISCUSSION TITLE
channel features (and democracy)

### DISCUSSION THREAD

**Discussion by rodriroberto24** [walofficial/wal-react-native#2] [Ideas]
მე არ ვიცი native, და ამიტომ მარტო ვერ ვიზამ, მაგრამ ვინც მზად ხართ 1-2 დღე დაუთმოთ, საინტერესო feature-ები მაქვს, რომელიც მარტივად შეიძლება ჩაინტეგრირდეს ამ ყველაფერში (AI-ს ხარჯებს მე ვფარავ, ისევე როგორც cloud db, etc.), უბრალოდ დამიკავშირდით, და ანონიმურობა დავიცვათ, სხვა მოთხოვნა არ მაქვს.

**Comment by vladimir-harkonnen** [comment]
I think it's easier and faster if you write the feature idea on the issue board where everyone can see it. I haven't found a template or guideline on how to request a feature, so I suggested one here: https://github.com/mntorg/mnt-react-native/issues/3

I also think it may be beneficial to keep discussions in English, in case some non-Georgian-speaking members also want to contribute.

**Reply by rodriroberto24** [reply]
Thanks for the comment. I suppose it might be a good idea for common feature suggestions, but in my case I have the whole application which I'd like to merge with MNT.
I don't want to expose everything unless I know there is somebody willing to do the work, and it seems to me that community is pretty small for now and there is not many (if anyone) willing to contribute.
